### DISCUSSION TITLE
Regarding designs

### DISCUSSION THREAD

**Discussion by digambar-t7** [yessGlory17/nft-marketplace#16] [General]
Hey there,
       I liked your project &  I want to contribute to it.  I want to know if you have prepared some layout designs for other pages as well so that I could create components of them.

**Comment by yessGlory17** [comment]
No, I was not prepared. I'm trying to finish building core components. I will have finished the core components by Sunday. If there is another issue that interests you, you can take the task. We especially need someone to help with the design.
